app/javascript/api/apiEndpoints.ts

```diff
@@ -1,4 +1,5 @@
-export const listing = (listingId) => `/api/v1/listings/${listingId}.json`
+export const listing = (listingId, preview = false) =>
+  `/api/v1/listings/${listingId}.json?force=${preview ? "true" : "false"}`
```

Comment:
We should generalize the implementation of the `force=true` param so that it works for all relevant endpoints, similar to how it is done in the Angular code.

We can do this by modifying the `get` function in our `apiService.ts` file:

```js
export const get = <T>(url: string, config?: AxiosRequestConfig): Promise<AxiosResponse<T>> => {
  const foo = { ...config.params }
  if (window.location.search.includes('preview=true')) {
    foo.force = 'true'
  }
  return axios.get(url, {...config, params: foo} )
}
```

Reply:
If you want to be a little less general, you could update the functions in `*ApiService.ts` files in a similar fashion.

Reply:
also, if you want to know which endpoints support the flag in Angular, you can grep for `forceRecache` in the repo